core/benches/banking_stage.rs

```diff
@@ -91,11 +93,49 @@ fn bench_consume_buffered(bencher: &mut Bencher) {
     let _unused = Blocktree::destroy(&ledger_path);
 }
 
-#[bench]
-fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
+fn make_accounts_txs(txes: usize, mint_keypair: &Keypair, hash: Hash) -> Vec<Transaction> {
+    let to_pubkey = Pubkey::new_rand();
+    let dummy = system_transaction::transfer(mint_keypair, &to_pubkey, 1, hash);
+    (0..txes)
+        .into_par_iter()
+        .map(|_| {
+            let mut new = dummy.clone();
+            let sig: Vec<u8> = (0..64).map(|_| thread_rng().gen()).collect();
+            new.message.account_keys[0] = Pubkey::new_rand();
+            new.message.account_keys[1] = Pubkey::new_rand();
+            new.signatures = vec![Signature::new(&sig[0..64])];
+            new
+        })
+        .collect()
+}
+
+fn make_programs_txs(txes: usize, hash: Hash) -> Vec<Transaction> {
+    let progs = 4;
+    (0..txes)
+        .into_iter()
+        .map(|_| {
+            let mut instructions = vec![];
+            let from_key = Keypair::new();
+            for _ in 1..progs {
+                let to_key = Pubkey::new_rand();
+                instructions.push(system_instruction::transfer(&from_key.pubkey(), &to_key, 1));
+            }
+            let mut new = Transaction::new_unsigned_instructions(instructions);
+            new.sign(&[&from_key], hash);
+            new
+        })
+        .collect()
+}
+
+enum TransactionType {
+    Accounts,
+    Programs,
+}
+
+fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
```

Comment:
A little cleaner would be to pass `transactions` in. Then you wouldn't need the `TransactionType` enum.

Reply:
Then i need to have another function to handle creating the transactions and the constants that derive the size.